lib/membrane_framerate_converter.ex

```diff
@@ -67,7 +68,7 @@ defmodule Membrane.FramerateConverter do
   def handle_caps(:input, %RawVideo{} = caps, _context, %{framerate: framerate} = state) do
     state = %{state | input_framerate: caps.framerate}
 
-    {{:ok, caps: {:output, %{caps | framerate: framerate}}}, state}
+    {{:ok, caps: {:output, %{caps | framerate: framerate}}}, %{state | caps_changed?: true}}
```

Comment:
There's a minor issue in the case of the first gap - `put_first_buffer` won't reset this flag so the first time `create_new_frames` is called it will only use the new buffer even though the caps haven't changed between buffers.

I see 2 options here:
1. Reset `caps_changed?` in `put_first_buffer`
2. Set `caps_changed?` conditionally, i.e. only when `state.last_buffer != nil`

Additionally, I'd group this state update with one in line 69